app/src/main/AndroidManifest.xml

```diff
@@ -22,7 +22,8 @@
         <activity
             android:name=".core.auth.AuthActivity"
             android:theme="@style/AppTheme.Main.Light"
-            android:label="${appName}">
+            android:label="${appName}"
+            android:windowSoftInputMode="stateHidden|adjustResize">
```

Comment:
This is restricting the layout to not re-render when adjusted or resized which will create problems when using tab layout or dual pane mode

Reply:
Can you please specify some conditions where it is raising the issues you mentioned which I can test?

Reply:
Currently, there is no problem. In future, there may be. We can't just enable these flags and change the resize behaviour just to fix a bug which is not caused by them. This is a workaround, not a fix

Reply:
I tested it with the dual pane mode, I think it is working fine.

Reply:
I just now tested it with dual pane mode, all looks good to me. And this behavior will only affect the layouts which are inherited by the AuthActivity. You can verify that the current approach is not just a workaround from the [documentation](https://developer.android.com/training/keyboard-input/visibility). Still, if you point me the exact issues, I can try to address them as well.

Reply:
Adding to the description of what's been done, `adjustResize` will only ensure that the system resizes your layout to the available space and `stateHidden` will only hide the soft keyboard till the user actually clicks on any of the field to input values.

Reply:
Let me know what conclusion you reached.

Reply:
Still, this is a workaround which is not needed for most activities, and not a fix

Add a todo in manifest acknowledging this

Reply:
@iamareebjamal Done.